flask_app/app.py

```diff
@@ -0,0 +1,53 @@
+from audioop import add
+from flask import Flask, request
+from flask_cors import CORS
+
+from mocks import SUPPORTED_EXCHANGES, MOCK_BALANCES_RESPONSE, MOCK_LOGIN_RESPONSE, MOCK_TRANSFER_RESPONSE
+
+app = Flask(__name__)
+CORS(app) # supports_credentials=True
+
+# another method/route to get exchanges we support
+
+@app.route("/")
+def home():
+    return "Flask is working!", 200
+
+@app.route("/api/v1/login", methods=["POST"])
+# log into the crypto exchange service
+def login():
+  args = request.args
+  exchange = args.get("exchange")
+  username = args.get("username")
+  password = args.get("password")
+  exchange_api = SUPPORTED_EXCHANGES[exchange]
```

Comment:
The server doesn't know about any other exchanges. It's only aware about itself.

Reply:
We aligned on storing the exchange APIs on Tally's side and keep this service general so that exchanges won't need to open a PR with us to integrate.